#! /bin/sh -e 
## ocamlobjinfo.dpatch by <luther@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Makefiles, configure scripts and other build stuff adapted for
## DP: debian package creation

if [ $# -ne 1 ]; then
    echo "`basename $0`: script expects -patch|-unpatch as argument" >&2
    exit 1
fi
case "$1" in
    -patch) patch -f --no-backup-if-mismatch -p0 < $0;;
    -unpatch) patch -f --no-backup-if-mismatch -R -p0 < $0;;	
    *)
	echo "`basename $0`: script expects -patch|-unpatch as argument" >&2
	exit 1;;
esac

exit 0
@DPATCH@
--- gdkPixbuf.ml.~1.9.~ 2004-03-24 01:49:00.000000000 +0100
+++ gdkPixbuf.ml        2004-08-03 16:51:38.592970653 +0200
@@ -67,7 +67,7 @@
   let nc = if has_alpha then 4 else 3 in
   let rowstride = match rowstride with None -> width * nc | Some r -> r in
   if bits <> 8 || rowstride < width * nc || width <= 0 || height <= 0
-  || Gpointer.length data < (rowstride * (height - 1) + width) * nc
+  || Gpointer.length data < rowstride * (height - 1) + width * nc
   then invalid_arg "GdkPixbuf.from_data";

